packages/styles/scss/components/card/index.scss

```diff
@@ -23,7 +23,7 @@
       height: 100%;
       display: flex;
       flex-direction: column;
-      padding: $carbon--spacing-05 $carbon--spacing-05 0;
+      padding: $carbon--spacing-05;
```

Comment:
I think what Jeff just mentioned It is happening because you are setting padding to all sides

Reply:
That code `padding: $carbon--spacing-05;` is already in the codebase, I edited it earlier to match the some specs but I reverted it and am going to speak to designer first